learningresources/models.py

```diff
@@ -129,6 +130,30 @@ class LearningResource(BaseModel):
     xa_nr_attempts = models.IntegerField(default=0)
     xa_avg_grade = models.FloatField(default=0)
     xa_histogram_grade = models.FloatField(default=0)
+    url_name = models.TextField(null=True)
+
+    @property
```

Comment:
It seemed like a good idea at the time but I can't remember why. I'll change it to a getter for clarity